test.sh

```diff
@@ -97,12 +93,41 @@ function setup_envs {
   export TEST_ASSET_ETCD=$kb_root_dir/bin/etcd
 }
 
+function is_installed {
+  if [ command -v $1 &>/dev/null ]; then
+    return 0
+  fi
+  return 1
+}
+
+function fetch_go_tools {
+  header_text "Checking for golangci-lint"
+  if ! is_installed golangci-lint; then
+    header_text "Installing golangci-lint"
+    GO111MODULE=off go get -d github.com/golangci/golangci-lint/cmd/golangci-lint
+    cd $(go env GOPATH)/src/github.com/golangci/golangci-lint
+    LATEST=v1.15.0
+    git checkout $LATEST
+    go install -ldflags="-X main.version=$LATEST" ./cmd/golangci-lint
```

Comment:
can't we just do `GO111MODULE=on go get github.com/golangci/golangci-ling/cmd/golangci-ling@v1.15.0` here/

Reply:
I think when I opened these PRs was before going all-in in modules so I wanted it to work for people not using them. I’ll change this. 

I’ve been a bit slow because of ongoing business travel, but I’ll also update the dockerfile for the test image to go in GCR.

Reply:
Err, oops, this is exactly what I said above >.< but yeah, as long as you’re fine with everything depending on modules there’s no problem.

Reply:
Flipped it back to use modules.